README.md

```diff
@@ -61,6 +61,17 @@ A data source backend plugin consists of both frontend and backend components.
    mage -l
    ```
 
+## Quickstart with docker:
+
+```
+docker run --it \
+      -p 3000:3000 \
+      --name=grafana \
+      -e "GF_INSTALL_PLUGINS=https://github.com/ecadlabs/tezos-grafana-datasource/releases/download/v0.0.1/ecad-labs-tezos-datasource-0.0.1.zip;ecad-labs-tezos-datasource" \
+      -e "GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS=ecad-labs-tezos-datasource" \
```

Comment:
This release is not available yet and the code is anticipating a release name of v0.0.1. In the future whoever doe the release will need to update these two lines with the new release version

Reply:
I feel like this is step is really easy to miss. Is there an API call to get the latest release of a repo?

Reply:
@danielelisi thanks for the feedback. I will look into it. I opened an issue for it here https://github.com/ecadlabs/tezos-grafana-datasource/issues/3